stock_picking_putaway_recompute/tests/test_recompute_putaway.py

```diff
@@ -266,14 +267,14 @@ def test_line_can_recompute(self):
             [False, False], move_lines_picking2.mapped("can_recompute_putaways")
         )
 
-    def test_recompute_putaway_packaging(self):
+    def test_recompute_putaway_package(self):
```

Comment:
Instead of modifying an existing test, I would have added a new one just testing what you modified/improved.
I wouldn't test everything in a single test.